src/test/java/bench/org/apache/zookeeper/BenchMain.java

```diff
@@ -0,0 +1,12 @@
+package org.apache.zookeeper;
```

Comment:
This file is missing apache license header. This triggers a -1 in last jenkins build.

Reply:
Will add it.